Minor cleanup and styling

Hlavtox opened the pull request:
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | See below
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | No
| How to test?      | See below
| Possible impacts? | None

### Changes and how to test
- Removed extra row > col. Check any page in BO for no visual change.
- Removed floats from grid Search & Reset buttons. See any grid for no visual change. Best page to test is Manufacturers. You should see no change, but it will help us in the future.
- Styled `/admin-dev/index.php/sell/catalog/products-v2/` a bit. Proper image size, stacked inputs, limited input widths, removed extra row >col.

Ping @NeOMakinG

![Výstřižek](https://user-images.githubusercontent.com/6097524/150786860-5d2f673c-47b7-4060-9d64-4ea213d7e4f2.JPG)

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/prestashop/prestashop/27443)
<!-- Reviewable:end -->

Hlavtox committed:
Minor cleanup and styling

Hlavtox committed:
Update _grid.scss

saulaski commented on admin-dev/themes/default/template/controllers/customer_threads/helpers/view/view.tpl:
```diff
@@ -80,13 +80,9 @@
-				</div>
+				{if !$first_message.id_employee}
+					{include file="./message.tpl" message=$first_message initial=true}
+				{/if}
```
The previous row/col is closed, so the message content doesn't require its own row/col ?

Hlavtox replied:
No :)

saulaski commented on admin-dev/themes/default/template/controllers/modules_positions/list_modules.tpl:
```diff
@@ -65,16 +65,12 @@
+						<input type="checkbox" id="hook_position"/>
+						{l s='Display non-positionable hooks' d='Admin.Design.Feature'}
+					</label>
+				</p>
```
Are you sure this content will behave as expected in the grid outside of any row or form-group ?

Hlavtox replied:
Yes dont worry

saulaski commented:
@QA Team Please test responsive display for smaller viewports.

Hlavtox commented:
@saulaski No changes there, it's not great, it switches to scroll on certain breakpoint, but we are preparing a mobile layout. ;-)

saulaski commented:
> @saulaski No changes there, it's not great, it switches to scroll on certain breakpoint, but we are preparing a mobile layout. ;-)

@Hlavtox I suppose your changes are correct indeed. But I also know from experience that Bootstrap grid might display unexpected results when content is put directly in container, without any row or column. Responsive testing cannot hurt :)

Hlavtox commented:
We deleted all hacks on the boostrap grid, now it works clean with no worries. 🚀

PS: I removed 5000 lines of this crap and all works good 😉 https://github.com/PrestaShop/PrestaShop/pull/25312

matthieu-rolland commented:
this is QA dev I guess ?  @PrestaShop/qa-team

NeOMakinG commented:
> this is QA dev I guess ? @PrestaShop/qa-team

Not really sure, these change can break some UI/UX interfaces

Robin-Fischer-PS commented:
I prefer a QA on this one, as @NeOMakinG said, it can break some UX. But maybe we'll need some help to test it :)

hibatallahAouadni reviewed (commented):
Hello @Hlavtox 

LGTM, except the new Product `List on experimental page` cause it's not the same as develop without PR.

- `List on experimental page` without PR
![image](https://user-images.githubusercontent.com/23308427/154242616-4a877baf-845f-45f6-8f1e-9b8c3e61150d.png)

- `List on experimental page` with PR
![image](https://user-images.githubusercontent.com/23308427/154242641-416fc208-be3d-4b31-a8f6-2b3fec716cb2.png)

Ping @PrestaShop/ux-team & @prestascott wdyt of these changes? can we approve it?

Thanks!

Hlavtox commented:
@hibatallahAouadni That was one of the points of the PR, to make it look like the legacy ;-) `Styled /admin-dev/index.php/sell/catalog/products-v2/ a bit. Proper image size, stacked inputs, limited input widths, removed extra row >col.`

hibatallahAouadni commented:
@Hlavtox I know :wink: we just need @prestascott confirmation/approval :slightly_smiling_face:

NeOMakinG commented:
@hibatallahAouadni so, the PR is fine, it's something that hasn't been finished on the page itself, it's not related to this PR

Also, I don't understand why we still merge PRs where the design is not yet finished to be integrated... `Let's do it later` should not be used everywhere...

hibatallahAouadni commented:
Ping @MatShir & @TristanLDD wdyt of this dilemma? :sweat_smile:

NeOMakinG commented:
Looks like @Hlavtox is the right one, but waiting for others then

saulaski commented:
> @hibatallahAouadni so, the PR is fine, it's something that hasn't been finished on the page itself, it's not related to this PR
> 
> Also, I don't understand why we still merge PRs where the design is not yet finished to be integrated... `Let's do it later` should not be used everywhere...

Totally agreed. The scope of the PR should be stricly fulfilled : no more, no less.

Hlavtox commented:
@saulaski @NeOMakinG Well, on one hand - yes, a PR should be complete. My PRs are not accepted without tests, why should be other PRs accepted with absolutely no responsivity?

On the other hand, when the BO devs don't have frontend skills, it's maybe better to do at least something, or the PR will take a year to get merged.

MatShir commented:
Looks good IMO, it applies the expected layout for the listing.

hibatallahAouadni commented:
Hello @Hlavtox 

Well then, as we have the PM approval, so LGTM and QA :heavy_check_mark: 

Thanks!

Progi1984 commented:
Thanks @Hlavtox and @hibatallahAouadni
